src/Hl7.Fhir.Specification.Tests/Snapshot/SnapshotGeneratorTest.cs

```diff
@@ -2800,9 +2800,9 @@ bool verifyElementBase(StructureDefinition original, StructureDefinition expande
                 else if (expanded.Kind == StructureDefinition.StructureDefinitionKind.Resource)
                 {
                     // [WMR 20190131] Fixed
-                    var baseDef = SnapshotGenerator.getBaseDefinition(expanded);
-                    bool isDerivedFromResource = baseDef == ModelInfo.CanonicalUriForFhirCoreType(FHIRAllTypes.Resource);
-                    bool isDerivedFromDomainResource = baseDef == ModelInfo.CanonicalUriForFhirCoreType(FHIRAllTypes.DomainResource);
+                    var baseDef = await _generator.getBaseDefinition(expanded);
```

Comment:
What's the difference between `SnapshotGenerator` and `_generator`? Is that the same instance?

Reply:
`getBaseDefinition` was earlier in this PR a static method. I changed it to an instance method.